### DISCUSSION TITLE
Next JS 13 + App Route

### DISCUSSION THREAD

**Discussion by HighError** [fief-dev/fief#247] [Q&A]
Does fief support next js 13 using App route, as the manual describes a method for pages route. If so, what would need to change for this?

And will there be problems with the client parts when you need to use "only client"?

**Comment by fief-bailiff** [comment]
Hail, @HighError 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Currently, I've not tested our integration with the new App Router. That said, since it mainly relies on [Middleware](https://nextjs.org/docs/app/building-your-application/routing/middleware), it'll probably work without any changes.

**Comment by DoWhileGeek** [comment]
I've been trying to figure around this all day, especially when it comes to server rendering, the lib seems broken for react 13

**Comment by DoWhileGeek** [comment]
At the very least, the fief docs for nextjs lead you off to unuseful places, in regards to nextjs 13.
